app/src/main/java/com/celzero/bravedns/util/Constants.kt

```diff
@@ -26,31 +25,31 @@ class Constants {
     companion object {
         // Download path and file names
         val DOWNLOAD_PATH = File.separator + "downloads" + File.separator
+
```

Comment:
fix naming (use approp suffixes and prefixes for _all_ constants) and leave one to two line comments on the constant vars?

Reply:
Done